libraries/apollo-compiler/api/apollo-compiler.api

```diff
@@ -40,13 +40,13 @@ public final class com/apollographql/apollo/compiler/CodegenMetadata {
 	public final fun plus (Lcom/apollographql/apollo/compiler/CodegenMetadata;)Lcom/apollographql/apollo/compiler/CodegenMetadata;
 }
 
-public synthetic class com/apollographql/apollo/compiler/CodegenMetadata$$serializer : kotlinx/serialization/internal/GeneratedSerializer {
+public final class com/apollographql/apollo/compiler/CodegenMetadata$$serializer : kotlinx/serialization/internal/GeneratedSerializer {
 	public static final field INSTANCE Lcom/apollographql/apollo/compiler/CodegenMetadata$$serializer;
-	public final fun childSerializers ()[Lkotlinx/serialization/KSerializer;
-	public final fun deserialize (Lkotlinx/serialization/encoding/Decoder;)Lcom/apollographql/apollo/compiler/CodegenMetadata;
+	public fun childSerializers ()[Lkotlinx/serialization/KSerializer;
+	public fun deserialize (Lkotlinx/serialization/encoding/Decoder;)Lcom/apollographql/apollo/compiler/CodegenMetadata;
 	public synthetic fun deserialize (Lkotlinx/serialization/encoding/Decoder;)Ljava/lang/Object;
-	public final fun getDescriptor ()Lkotlinx/serialization/descriptors/SerialDescriptor;
-	public final fun serialize (Lkotlinx/serialization/encoding/Encoder;Lcom/apollographql/apollo/compiler/CodegenMetadata;)V
+	public fun getDescriptor ()Lkotlinx/serialization/descriptors/SerialDescriptor;
+	public fun serialize (Lkotlinx/serialization/encoding/Encoder;Lcom/apollographql/apollo/compiler/CodegenMetadata;)V
 	public synthetic fun serialize (Lkotlinx/serialization/encoding/Encoder;Ljava/lang/Object;)V
 	public fun typeParametersSerializers ()[Lkotlinx/serialization/KSerializer;
```

Comment:
Lots of changes in this file because I downgraded the languageVersion to 1.9 to keep our Gradle 8 compatibility. 

See also https://github.com/Kotlin/kotlinx.serialization/issues/2108